src/helpers/LongTitelsTruncate.js

```diff
@@ -0,0 +1,7 @@
+
+
+function LongTitlesTruncate(title) {
+    return title.slice(0, 100).concat("...");
```

Comment:
Nu komen er altijd puntjes achter te staan in plaats van alleen bij namen die langer zijn dan 100 karakters